### ISSUE TITLE
recognize.py Working!

### ISSUE THREAD

**Issue by EN10** [buriburisuri/speech-to-text-wavenet#49]
If **Dependencies** are installed as described then it works.
Seems to brake with newer dependencies installed.
Used: `pip freeze` to see which version installed.
If a newer version is installed then uninstall: `sudo pip uninstall sugartensor`
Then install correct version: `sudo pip install sugartensor==0.0.1.9`
To install correct version of tensorflow: `sudo pip install https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.11.0-cp27-none-linux_x86_64.whl`

**Comment by abc1199281** [comment]
It seems awesome!!! I will give it a try. Excuse me, do you know the dependencies corresponding to the Tensorflow version 0.12?

**Comment by EN10** [comment]
I been testing a few variations, this is the only one that works for me so far.
I think you can probably update some of the packages and it will still work.
I think it is sensitive to sugartensor and possibly also tensorflow versions.

**Comment by gbrouwer** [comment]
Thanks, EN10, I'll give it a try as well!

**Comment by gvoysey** [comment]
@EN10 thanks for this. 

do you know if there are any side-effects of using a CUDA-enabled version of tf?

**Comment by EN10** [comment]
@gvoysey I have not used CUDA as I'm not training. I'm using the pretrained model given by @buriburisuri which I have extracted here: https://github.com/EN10/STT/tree/master/asset/train/ckpt

**Comment by gbrouwer** [comment]
I can confirm the suggested dependencies work on my Mac. train.py is now running.

**Comment by EN10** [comment]
I got it working on Bash on Ubuntu on Windows 10 and Ubuntu in Docker.

**Comment by gbrouwer** [comment]
I can also verify it is working on my Ubuntu 14.04 Amazon EC2 box, with GPU CUDNN support.

**Comment by gvoysey** [comment]
@gbrouwer where did you get a CUDNN supported version of tf 0.11 ? I need one built against cuda 8.

**Comment by gbrouwer** [comment]
@gvoysey : I used this link with installation instructions for TF0.11: 
https://www.tensorflow.org/versions/r0.11/get_started/os_setup

**Comment by EN10** [comment]
@gvoysey @gbrouwer 
As this is outputting a character at a time and there is no language model I find the accuracy is not so good. Wonder if you get any better success?
This looks interesting: https://github.com/mozilla/DeepSpeech

**Comment by gvoysey** [comment]
Bingo.  thanks much @gbrouwer

**Comment by sidh104** [comment]
Thanks @EN10

**Comment by EN10** [comment]
Your Welcome :) @sidh104

**Comment by marshonhuckleberry** [comment]
os, py version?
